tests/test_basic_ops_dict.py

```diff
@@ -119,50 +118,249 @@ def test_subscript_set_with_dot_warning_suppressed(recwarn: Any, mocker: Any) ->
     assert len(recwarn) == 0
 
 
-def test_pretty_dict() -> None:
-    c = OmegaConf.create(dict(hello="world", list=[1, 2]))
-    expected = """hello: world
-list:
-- 1
-- 2
-"""
-    assert expected == c.pretty()
-    assert OmegaConf.create(c.pretty()) == c
+def test_instantiate_config_fails() -> None:
+    with pytest.raises(TypeError):
+        BaseContainer()  # type: ignore
```

Comment:
This is not a real change, you are just moving it around in the file.
these kind of changes are making it really hard to review the diff.
Please fix your diff. at that point it might be easier to revert the changes to the logs and move just the correct functions to the new file.